ulas/assetlist.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "ULAS",
+  "assets": [
+    {
+      "description": "WEB3 twitter",
+      "denom_units": [
+        {
+          "denom": "uulas",
+          "exponent": 0
+        },
+        {
+          "denom": "ulas",
+          "exponent": 6
+        }
+      ],
+      "base": "uulas",
+      "name": "ULAS Network",
+      "display": "ULAS",
+      "symbol": "ULAS",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/ulasnetwork/chain-config/main/ulas/images/logo.png"
```

Comment:
Please re-point this URI to the local repo

Reply:
Done